lib/core/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import { App as Application, Plugin } from "vue";
+import * as components from "./components/index";
```

Comment:
I see you are using both relative path and absolute path. I suggest using only absolute path for consistent.